### PR TITLE
Create delete method

### LINKED ISSUE #124: Add Delete endpoint for the CleaningEventController
## Description

Implement a new new `Delete` method for the `CleaningEventController`, which should take the `ObjectIdentifyRequest` with the `FromQuery` attribute.

## Steps to take

1. Take a look at the delete implementation found in the `PollutedLocationController`. You can use it as a reference. 
2. Create required maps for cleaning event mapping profile.
3. Write unit tests to cover the implemented functionality in the controller, mapper and other parts if modified.

## Acceptance criteria

You can delete an instance of CleaningEvent using a Delete request in swagger, with the endpoint /Delete/.
The implementation is covered with Unit tests.

### PR DESCRIPTION
**Pull request by BaVilius**
Created a Delete method for Cleaning Events

- Implemented method
- Created a mapper for ObjectIdentifyRequest To CleaningEvent
- Written unit tests

### CODE REVIEW

File: Apsitvarkom.Api/Controllers/CleaningEventController.cs [hunk 1]
```diff
@@ -66,11 +66,48 @@ public async Task<ActionResult<CleaningEventResponse>> GetById([FromQuery] Objec
             return StatusCode(StatusCodes.Status500InternalServerError);
         }
 
-        if (cleaningEvent is null) return NotFound($"Polluted location with the specified id '{cleaningEventIdentifyRequest.Id}' was not found.");
+        if (cleaningEvent is null) return NotFound($"Cleaning event with the specified id '{cleaningEventIdentifyRequest.Id}' was not found.");
```
Comment (6ar8nas) [inline Apsitvarkom.Api/Controllers/CleaningEventController.cs:69]: hehe, nice:D missed it myself

File: Apsitvarkom.Models/Mapping/CleaningEventProfile.cs [hunk 2]
```diff
@@ -17,7 +17,8 @@ public CleaningEventProfile()
 
     private void MapRequests()
     {
-
+        CreateMap<ObjectIdentifyRequest, CleaningEvent>(MemberList.None).
```
Comment (6ar8nas) [inline Apsitvarkom.Models/Mapping/CleaningEventProfile.cs]: Kind of a styling related nitpick, but looking at where to place the dot (before or after the line break) or similar questions (for example 
```
a
+ b
``` 
or 
```
a + 
b
```
I usually refer to [this](https://google.github.io/styleguide/javaguide.html#s4.5.1-line-wrapping-where-to-break) article. That's the usual practices used at Google (of course in a work scenario, you would instead keep up with what is around you or a preferred coding style). 
That article suggests having the dot in the new line [4.5.1.1], so maybe let's just sliiightly adjust this?

File: Apsitvarkom.Api/Controllers/CleaningEventController.cs (hunk 1 above)
Comment (6ar8nas) [reply Apsitvarkom.Api/Controllers/CleaningEventController.cs:69]: (feel free to resolve this one on your own after noticing)

File: Apsitvarkom.UnitTests/Api/Controllers/CleaningEventControllerTests.cs [hunk 3]
```diff
@@ -210,4 +210,113 @@ public async Task GetById_RepositoryThrows_Status500InternalServerErrorReturned(
...
+        Assert.That(actionResult, Is.TypeOf<BadRequestObjectResult>());
+        var result = actionResult as BadRequestObjectResult;
+
+        Assert.That(result, Is.Not.Null);
+        Assert.That(result.StatusCode, Is.EqualTo(StatusCodes.Status400BadRequest));
+        Assert.That(result.Value, Is.Not.Null.And.Not.Empty);
+    }
+
+    [Test]
+    public async Task Delete_RepositoryThrowsAcquiringPollutedLocation_Status500InternalServerErrorReturned()
```
Comment (6ar8nas) [inline Apsitvarkom.UnitTests/Api/Controllers/CleaningEventControllerTests.cs]: some names use `PollutedLocation`, however we are now testing `CleaningEvent`s here. let's adjust that

File: Apsitvarkom.UnitTests/Models/Mapping/CleaningEventMappingTests.cs [hunk 4]
```diff
@@ -23,7 +23,18 @@ public void OneTimeSetUp()
     }
 
     #region Request mappings
+    [Test]
+    public void ObjectIdentifyRequestToCleaningEventResponse()
```
Comment (6ar8nas) [inline Apsitvarkom.UnitTests/Models/Mapping/CleaningEventMappingTests.cs]:
```suggestion
    public void ObjectIdentifyRequestToCleaningEvent()
```
we are mapping not to `CleaningEventResponse` here

**Review by 6ar8nas — CHANGES_REQUESTED** [review]
slight adjustments. apart from that, looks good

File: Apsitvarkom.Models/Mapping/CleaningEventProfile.cs (hunk 2 above)
Comment (BaVilius) [reply Apsitvarkom.Models/Mapping/CleaningEventProfile.cs]: Sure

File: Apsitvarkom.UnitTests/Api/Controllers/CleaningEventControllerTests.cs (hunk 3 above)
Comment (BaVilius) [reply Apsitvarkom.UnitTests/Api/Controllers/CleaningEventControllerTests.cs]: Whoops

**Review by 6ar8nas — APPROVED** [review]

**Pull request merged**